lib/ledger_sync.rb

```diff
@@ -38,6 +39,8 @@
 require 'ledger_sync/result'
 require 'ledger_sync/adaptors/operation'
 require 'ledger_sync/adaptors/contract'
+require 'ledger_sync/adaptors/response'
```

Comment:
@jozefvaclavik assuming you like this pattern, it'd be nice to wrap our other requests in these objects for consistency.